app/views/course/assessment/answer/programming/_annotation_topic.html.slim

```diff
@@ -1,2 +1,3 @@
-= render partial: 'course/discussion/topic', object: annotation_topic,
-         locals: { post_partial: 'course/assessment/answer/programming/annotation_post' }
+- unless annotation_topic.posts.empty?
```

Comment:
Would changing the line [highlighted here](https://github.com/Coursemology/coursemology2/blob/master/app/helpers/course/assessment/answer/programming_helper.rb#L39) solve that?

```ruby
# Old
    programming_answer_file.annotations.each do |annotation|
      insert_annotation_block(document, annotation)
    end

# New
   programming_answer_file.annotations.includes(:posts).each do |annotation|
      insert_annotation_block(document, annotation)
    end
```